website/content/docs/troubleshooting.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Troubleshooting Guide
```

Comment:
Can we give this doc a more specific title? We have another called 'Topology Troubleshooting Guide' for example. We should give this one a name that distinguishes it from other troubleshooting guides and makes it's scope clear.